README.md

```diff
@@ -1,125 +1,118 @@
+- [XDG Base Directory](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) friendly.
+- `.node-version` and `.nvmrc` support. ✅
+- <kbd>Tab</kbd>-completable seamless shell integration.
+- No dependencies, no setup, no clutter—it just works.
+  <!-- - Hot symlink switching—absolute speed unlocked. -->
+    <!-- - Automatic version switching on `$PWD` change. -->
 
 ## Installation
 
-Install with [Fisher](https://github.com/jorgebucaran/fisher) (recommended):
+Install with [Fisher](https://github.com/jorgebucaran/fisher):
 
 ```console
 fisher install jorgebucaran/nvm.fish
 ```
 
-<details>
-<summary>Not using a package manager?</summary>
+## Quickstart
 
-###
+Install the latest Node release and start using it.
 
-Copy [`conf.d/nvm.fish`](conf.d/nvm.fish), [`functions/nvm.fish`](functions/nvm.fish), and [`completions/nvm.fish`](completions/nvm.fish) to your fish configuration directory preserving the directory structure.
+```console
+nvm install latest
+```
 
-```fish
-set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
+Install the latest [LTS](https://github.com/nodejs/Release) (long-term support) Node release.
 
-for i in conf.d functions completions
-  curl https://git.io/$i.nvm.fish --create-dirs -sLo $XDG_CONFIG_HOME/fish/$i/nvm.fish
-end
+```console
+nvm install lts
 ```
 
-To uninstall nvm, just run:
+Install an older LTS release by codename.
 
-```
-rm -f $XDG_CONFIG_HOME/fish/{conf.d,functions,completions}/nvm.fish && emit nvm_uninstall
+```console
+nvm install carbon
 ```
 
-</details>
+> Installs `8.16.2`, the latest release of the Carbon LTS line.
 
-## Quickstart
-
-Download and switch to the latest Node.js release.
+Or install a specific version of Node.
 
 ```console
-nvm use latest
+nvm install v12.9.1
 ```
 
-> **Note:** This downloads the latest Node.js release tarball from the [official mirror](https://nodejs.org/dist), extracts it to <code>[\$XDG_CONFIG_HOME](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables)/nvm</code> and modifies your `$PATH` so it can be used immediately. Learn more about the Node.js release schedule [here](https://github.com/nodejs/Release).
+> Supports full or partial version numbers, starting with an optional "v".
 
-Download and switch to the latest LTS (long-term support) Node.js release.
+The `nvm install` command activates the specified Node version only in the current environment. If you want to set the default version for new shells use:
 
-```console
-nvm use lts
+```fish
+set -U nvm_default_version v12.9.1
 ```
 
-You can create a `.nvmrc` file in the root of your project (or any parent directory) and run `nvm` to use the version in it. `nvm` will try to find the nearest `.nvmrc` file, traversing the directory tree from the current working directory upwards.
+Activate a version you've already installed.
 
 ```console
-node -v > .nvmrc
-nvm
+nvm use lts
```

Comment:
Will this fail if the version trying to be used does not exist? Not sure if it should or not i.e. what is the best option.

Reply:
Yes, it will fail with an error message, just like the original nvm.